.github/workflows/pr-checks.yaml

```diff
@@ -181,7 +181,12 @@ jobs:
 
           eval $(minikube docker-env)
           make release-manifests
+          make deploy-cert-manager
```

Comment:
we can combine this with previous line:

```
make deploy-cert-manager release-manifest
```

Reply:
Kept both separated since `deploy-cert-manager` is paired with a `kubectl rollout status`. This might change though if the rollout status is moved to the Makefile's target `deploy-cert-manager`.

Reply:
Done in 176d73f10d5047ed3b2383d4136d23e693c874a5.